src/main/java/de/tum/in/www1/artemis/service/LectureService.java

```diff
@@ -30,6 +33,8 @@
 @Service
 public class LectureService {
 
+    private final Optional<PyrisWebhookService> pyrisWebhookService;
```

Comment:
Since this is optional can you put it at the bottom of this list (also in the constructor)